index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+  <title>Kartikay's Portfolio</title>
+</head>
+
+<body class="main-block">
+  <header class="head-menu">
+    <nav class="nav-mobile">
+      <a class="logo" href="#">My Logo</a>
+      <i class="fa fa-bars"></i>
```

Comment:
In the further development of the project, the hamburger button will hold some actions. When clicked, it will display the mobile navigation menu.

- [ ]  It is recommended here to add to the hamburger button an `a` tag.

Reply:
Thanks for the suggestion. I have done the required changes.